### PR TITLE
Merging new styles and documentation

### PR DESCRIPTION
**Pull request by gvwilson**
This PR brings in all of the 2016-06 template, along with documentation on building lessons.

### CODE REVIEW

File: _episodes/05-history.md [hunk 1]
```diff
@@ -1,23 +1,30 @@
 ---
-layout: page
-title: Version Control with Git
-subtitle: Exploring History
-minutes: 25
+title: Exploring History
+teaching: 25
+exercises: 0
+questions:
```
Comment (iglpdc) [inline _episodes/05-history.md]: Which is the difference between "questions" and "objectives"?

File: index.md [hunk 2]
```diff
@@ -43,35 +43,9 @@ as well.  And it isn't just for software: books,
...
 to be shared can and should be stored in a version control system.
 
-> ## Prerequisites {.prereq}
+> ## Prerequisites
 >
 > In this lesson we use Git from the Unix Shell.
 > Some previous experience with the shell is expected,
 > *but isn't mandatory*.
-
-> ## Getting ready {.getready}
```
Comment (iglpdc) [inline index.md]: Is the "Getting ready" callout going somewhere else? I think we should keep (in this way or another) to be sure everybody is starting from the same setup.

Comment (iglpdc) [inline index.md]: Sorry, I see it now: it's cool.

**Comment by iglpdc** [comment]
It looks good to me.

**Comment by iglpdc** [comment]
@gvwilson when do you think you'll be done? There is a couple of small PR almost ready to be merge in, but maybe I'll wait to merge this one, so we don't have to deal with this monster much.

**Comment by iglpdc** [comment]
Also, we have to add #309. (I can do that after we merge this one).

**Comment by gvwilson** [comment]
Hi Ivan,
Should merge tomorrow.
Cheers,
Greg

**Pull request merged**